californium-core/src/test/java/org/eclipse/californium/core/test/ClientAsynchronousTest.java

```diff
@@ -157,6 +157,46 @@ public void testAsyncObserveTriggersOnLoad() throws Exception {
 		Thread.sleep(50);
 		assertThat(receivedNotifications.get(), is(3));
 	}
+	
+	/**
+	 * Test case for CoapClient.observe(Request request, CoapHandler handler).
+	 * 
+	 * @author Praful Bhatnagar (prafulbhatnagar@gmail.com)
```

Comment:
please remove the `@author` tag. We can always see who added which code by means of the commit log